packages/cli/src/commands/generate/auth/auth.js

```diff
@@ -20,7 +20,8 @@ const SUPPORTED_PROVIDERS = fs
 // returns the content of index.js with import statements added
 const addWebImports = (content, imports) => {
   const importStatements = imports.map((imp) => {
-    return `import ${imp.import} from '${imp.from}'`
+    const importWithFrom = null !== imp.import ? `${imp.import} from` : ''
```

Comment:
I don't know that we do this reverse expectation syntax anywhere else, `null !== imp.import`. How do you feel about that @peterp?

Reply:
I think this is what they called yoda conditions? https://en.wikipedia.org/wiki/Yoda_conditions

Reply:
Busted, 10 years on the php side of the force ( 10 years and leaving to JS! )
```suggestion
const importWithFrom = imp.import === null ? '' : `${imp.import} from`;
```
Any better? Or should I go for an if statement?

Reply:
You're using _the force_, I'm also from the PHP world!

Looks good to me.